gulp-ng-config.js

```diff
@@ -148,7 +148,7 @@ function gulpNgConfig (moduleName, configuration) {
       });
     }
 
-    file.path = gutil.replaceExtension(file.path, '.js');
+    file.path = gutil.replaceExtension(configuration.fileName || file.path, '.js');
```

Comment:
It looks like previously we had the entire file _path_, what happens if this is given a filename without any path? for example 'foo.js' vs './foo.js'